hoot-core/src/main/cpp/hoot/core/io/HootApiDb.cpp

```diff
@@ -353,7 +353,24 @@ void HootApiDb::dropTable(const QString& tableName)
 
   // inserting strings in this fashion is safe b/c it is private and we closely control the table
   // names.
-  QString sql = QString("DROP TABLE IF EXISTS %1").arg(tableName);
+  QString sql = QString("DROP TABLE IF EXISTS %1 CASCADE;").arg(tableName);
```

Comment:
Added `CASCADE` to the `DROP TABLE` because of the foreign key constraints on those tables that, in some cases, didn't allow the tables to be deleted.